src/frontend/src/components/common/InputText/InputText.tsx

```diff
@@ -81,8 +81,12 @@ const InputText = ({
           required={required}
           disabled={isDisabled}
         />
-      ) : (
+      ) : value ? (
         <p>{value}</p>
+      ) : (
+        <p>
+          <i>Not answered</i>
```

Comment:
You may need to add this Not answered to rest of your common components too. Like Checkboxes or Dropdown

Reply:
Done.